locales/FACTER.pot

```diff
@@ -6,7 +6,7 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: FACTER 4.0.0\n"
```

Comment:
No, we had temporarily thought we were going to ship a Facter 4, then decided against it. It appears this file missed getting updated.